easybuild/easyconfigs/s/simint/simint-0.7-foss-2021b-lmax-5-vec-avx-psi4.eb

```
@@ -0,0 +1,51 @@
+# For the various build options see here:
+# https://www.bennyp.org/research/simint/README.txt
+# lmax is set to 5 which is widely used it seems
+# vec is set to avx, which should work on most platforms
+# This might be changed to other values, if required
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'simint'
+version = '0.7'
+local_lmax = '5'
+local_vec = 'avx'
+
+# custom configuration, to be used as dependency for PSI4
+versionsuffix = '-lmax-%s-vec-%s-psi4' % (local_lmax, local_vec)
+
+homepage = 'https://www.bennyp.org/research/simint/'
+description = """Simint is a vectorized implementation of the Obara-Saika (OS) 
+method of calculating electron repulsion integrals. Speedup is gained by 
+vectorizing the primitive loop of the OS algorithm, with additional vectorization 
+and optimizations left to the compiler."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
```

this can be dropped to `GCCcore` since it doesn't depend on blas/lapack/mpi (remember that the filename would need updating too)
```suggestion
toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
```

---

since this software is performance critical, I think it's better to use `GCC` toolchain, so we can have an intel version as well.

---

Done. Do you want me to do an `Intel` version as well or shall we wait until it is needed?

---

thanks!

> Do you want me to do an `Intel` version as well or shall we wait until it is needed?

I'll leave that up to you. it's nice to have both to compare the performance, but then of course someone has to do the comparison.